tests/testthat/_snaps/chuck.md

```diff
@@ -0,0 +1,8 @@
+# trying to chuck NULL raises errors
+
+    Code
+      chuck(NULL, "a")
+    Condition
+      Error:
+      ! Index 1 is attempting to pluck from an unnamed vector using a string name
```

Comment:
It's a bit confusing to refer to `NULL` as an unnamed vector but I guess it might be cumbersome to do better here.